example_project/example/models/example/http_events_bytes_sent.sql

```diff
@@ -1,4 +1,4 @@
-{{ config(primary_key="method") }}
+{{ config(primary_keys=["method"]) }}
```

Comment:
I'm questioning the name here a bit. I think it's _one_ primary key, composed of _many_ fields. But `primary_keys` reads like there are multiple primary keys.

What do you think about using either `primary_key` as before, or something like `primary_key_fields`?

In our API, we use just `primary_key` and it takes an array as well, so I'm leaning that way for consistency's sake.

Reply:
this is spot on. I'll make the change